.circleci/config.yml

```diff
@@ -224,10 +224,10 @@ jobs:
       - checkout
       - restore_cache: *restore_cache
       - run: *gh-config
-      - run: *install
```

Comment:
Having this as an alias is causing install issues so I'm updating it to a normal `npm install` in the job.